setup.py

```diff
@@ -22,7 +22,7 @@ def read(fname):
 
 setup(
     name='ecs-cluster',
-    version='1.5.3',
+    version='1.5.4',
```

Comment:
Shouldn't this be 1.6.0?

Reply:
I had the same internal question :) Since we both asked, I'll change it.